config/analytics_blocklist.yml

```diff
@@ -578,3 +578,177 @@
   - data
   - created_at
   - updated_at
+# NPQ Registration Source Models
+  :schools:
+  - la_code
+  - la_name
+  - establishment_number
+  - establishment_status_code
+  - establishment_status_name
+  - close_date
+  - last_changed_date
+  - address_1
+  - address_2
+  - address_3
+  - town
+  - county
+  - easting
+  - northing
+  - region
+  - country
+  - establishment_type_code
+  - establishment_type_name
+  - high_pupil_premium
+  - postcode_without_spaces
+  - number_of_pupils
+  - eyl_funding_eligible
+  - phase_type
+  - phase_name
+  :local_authorities:
+  - ukprn
+  - address_1
+  - address_2
+  - address_3
+  - town
+  - county
+  - postcode
+  - postcode_without_spaces
+  - high_pupil_premium
+  :lead_providers:
```

Comment:
I think we will end up with some NPQ models in our BigQuery here; need to check if the model class is sent as part of the event so that we can differentiate as it doesn't look like dfe-analytics will handle two models with the same class name in different namespaces here (as its using the table name, which is the same).